sdk/program/src/stake/state.rs

```diff
@@ -335,7 +335,7 @@ impl Default for Delegation {
             stake: 0,
             activation_epoch: 0,
             deactivation_epoch: std::u64::MAX,
-            warmup_cooldown_rate: 0.0,
+            warmup_cooldown_rate: DEFAULT_WARMUP_COOLDOWN_RATE,
```

Comment:
this needs fixed in master too, right?  let's just do it there first and backport for the sake of consistent history

Reply:
I was more thinking that it would be bad to have a release branch in a known broken state in between PRs, but I guess consistent history is probably more important in a world of squashed commits.

Gerrit would handle this better :trollface:

Reply:
omg since when does gh have :trollface: ?

Reply:
:trollface: